tardis/energy_input/gamma_packet_loop.py

```diff
@@ -303,22 +278,20 @@ def gamma_packet_loop(
                     packet.nu_cmf,
                     packet.nu_rf,
                     packet.energy_cmf,
-                    lum_rf,
+                    luminosity,
                     packet.energy_rf,
                     packet.shell,
                 ]
             )
 
-    print("Escaped packets:", escaped_packets)
-    print("Scattered packets:", scattered_packets)
+    print("Number of escaped packets:", escaped_packets)
+    print("Number of scattered packets:", scattered_packets)
```

Comment:
See note about print statements above

Reply:
I think these `print` statements are useful diagnostics. I am not sure how to use logging here.